cdap-ui/app/cdap/components/EntityListView/ListView/index.js

```diff
@@ -42,6 +47,36 @@ export default class HomeListView extends Component {
     });
   }
 
+  onClick(entity) {
+    let activeEntity = this.state.list.filter(e => e.id === entity.id);
+    if (activeEntity.length) {
+      this.setState({
+        activeEntity: activeEntity[0]
+      });
+    }
+    if (this.props.onEntityClick) {
+      this.props.onEntityClick(entity);
+    }
+  }
+
+  filtersApplied() {
+    return this.props.activeFilter.length > 0 && this.props.activeFilter.length < this.props.filterOptions.length;
```

Comment:
Typing in a search term with 0 results could show a message but unchecking even one option from the dropdown will show another message right? Might not be consistent